packages/core/src/mastra/index.ts

```diff
@@ -444,6 +464,12 @@ Do:
         this.#vectors?.[key]?.__setLogger(this.#logger);
       });
     }
+
+    if (this.#mcpServers) {
+      Object.keys(this.#mcpServers).forEach(key => {
+        this.#mcpServers?.[key]?.__setLogger(this.#logger);
```

Comment:
move this to the for each above

Reply:
all the other loggers get set this way though

Reply:
Oh yeah sorry it's a different function thought it was inside the same